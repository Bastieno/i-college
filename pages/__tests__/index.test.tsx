import React from 'react';
import { mount, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Index from '../index';

describe('Index page', () => {
  it('should have the App component', () => {
    const wrapper = mount(<Index />);

    expect(wrapper.find('App')).toHaveLength(1);
  });

  it('should render without crashing', () => {
    const wrapper = shallow(<Index />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
