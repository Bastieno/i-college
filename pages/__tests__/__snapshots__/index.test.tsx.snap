// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should render without crashing 1`] = `<App />`;
